tests/appsec/iast/sink/test_xcontent_sniffing.py

```diff
@@ -0,0 +1,63 @@
+# Unless explicitly stated otherwise all files in this repository are licensed under the the Apache License Version 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2021 Datadog, Inc.
+
+from utils import context, coverage, released, missing_feature
+from .._test_iast_fixtures import SinkFixture
+
+
+@coverage.basic
+@released(dotnet="?", golang="?", php_appsec="?", python="?", ruby="?", nodejs="?")
+@released(
```

Comment:
See https://github.com/DataDog/system-tests/actions/runs/6081906417/job/16509643708?pr=1539#step:4:33

You must use only the manifest file to declare this

Reply:
Done